.github/workflows/ci.yml

```diff
@@ -0,0 +1,83 @@
+name: Alfresco Java SDK CI
+
+on:
+  push:
+    branches:
+      - develop
+      - release/**
+      - dev-**
+  pull_request:
+    branches:
+      - develop
+      - release/**
+      - dev-**
+  workflow_dispatch:
+
+env:
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  CONTENT_SERVICE_SECURITY_BASICAUTH_PASSWORD: ${{ secrets.CONTENT_SERVICE_SECURITY_BASICAUTH_PASSWORD }}
+  CONTENT_SERVICE_SECURITY_BASICAUTH_USERNAME: ${{ secrets.CONTENT_SERVICE_SECURITY_BASICAUTH_USERNAME }}
+  CONTENT_SERVICE_URL: ${{ secrets.CONTENT_SERVICE_URL }}
+  ENVIRONMENT_APA_HOST: apadev-apa.envalfresco.com
+  ENVIRONMENT_HOST: apadev.envalfresco.com
+  ENVIRONMENT_APP: simpleapp
+  AGS_VERSION: 14.9
+  ACS_VERSION: 7.1.0.1
+
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.35.0
+
+  build_and_verify:
+    name: "Build and Verify"
+    runs-on: ubuntu-latest
+    needs:
+      - pre-commit
+    env:
+      MAVEN_CLI_OPTS: >
+        source:jar -B -q -e -fae -V -DinstallAtEnd=true -U -Ddoclint=none
+        -Denvironment.host=${ENVIRONMENT_HOST}
+        -Denvironment.apa.host=${ENVIRONMENT_APA_HOST}
+        -Denvironment.application.name=${ENVIRONMENT_APP}
+        -Dags.version=${AGS_VERSION}
+        -Dacs.version=${ACS_VERSION}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.0
+      - name: "Build"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: mvn install $MAVEN_CLI_OPTS -DskipTests=true -Dmaven.javadoc.skip=true
+      - name: "Run tests"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: mvn verify $MAVEN_CLI_OPTS -Dlogging.root.level=off -Dspring.main.banner-mode=off
+
+  publish:
+    name: "Publish"
+    runs-on: ubuntu-latest
+    if: >
+      github.event_name != 'pull_request'
+    needs:
+      - build_and_verify
+    env:
+      MAVEN_CLI_OPTS: >
+        source:jar -B -q -e -fae -V -DinstallAtEnd=true -U -Ddoclint=none
+        -DaltReleaseDeploymentRepository=alfresco-public::default::https://artifacts.alfresco.com/nexus/content/repositories/releases
+        -DaltSnapshotDeploymentRepository=alfresco-public-snapshots::default::https://artifacts.alfresco.com/nexus/content/repositories/snapshots
```

Comment:
I'd assume @gionn review comment also applies to these.

Reply:
done